src/ApolloClient.re

```diff
@@ -0,0 +1,128 @@
+open ReasonApolloTypes;
```

Comment:
would be amazing if this would be moved to another package so reason-apollo and reason-apollo-hooks could share the same ApolloClient definitions

Reply:
Probably better to have a single package, because the hooks are now also part of the same `@apollo/client` package.